ui/objects/semantic-ui/js/objects/search/components/ResultsListItem.jsx

```diff
@@ -89,54 +78,62 @@ export const ResultsListItemComponent = ({ result, appName }) => {
     "metadata.restorationWork.restorer",
     "<no data>"
   );
-  const desc = _get(
-    result,
-    "metadata.restorationWork.abstract",
-    "<no data>"
-  );
 
   const created = _get(result, "created", "<no data>");
+  const creationDate = new Date(created).toLocaleDateString();
 
+  const restDescription = _get(result, "metadata.restorationWork.abstract", "");
 
+  const objDescription = _get(
+    result,
+    "metadata.restorationObject.description",
+    ""
+  );
 
+  function stripHtmlTags(html) {
+    const tmpElement = document.createElement("div");
```

Comment:
No document.createElement allowed, we have React to manage DOM elements :)

Reply:
Additionally, setting innerHTML to user input is a bad security practice: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#replacing_the_contents_of_an_element